_client/src/Ui/Dialogs/DialogManager.cpp

```diff
@@ -99,6 +101,20 @@ void DialogManager::showMessage(const QString &title, const QString &message, QM
     m_messageDialog->setIcon(type);
     m_messageDialog->setWindowTitle(title);
     m_messageDialog->setText(message);
-    m_messageDialog->addButton(QMessageBox::Ok);
     m_messageDialog->exec();
+}
+
+void DialogManager::onLoginSuccess()
+{
+    m_loginDialog->hide();
+    showMessage(tr("Message"), tr("Login successfull"), QMessageBox::Information);
+}
+
+void DialogManager::onRegistrationSuccess()
+{
+    m_registrationDialog->hide();
+    m_loginDialog->m_usernameField->setText(m_registrationDialog->m_usernameField->text());
```

Comment:
Do not break encapsulation. 
Also, consider using already existing method `LoginDialog::showWithPresetData`